resources/queries/targetedms/QCInstrumentSummary.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+    sf.InstrumentId.model AS InstrumentName,
+    sf.InstrumentSerialNumber AS SerialNumber,
+    MIN(sf.AcquiredTime) AS StartDate,
+    MAX(sf.AcquiredTime) AS EndDate,
+    COUNT(DISTINCT sf.ReplicateId) AS NoOfReplicates,
+    null AS QCFolders
```

Comment:
Instead of null, have this use sf.InstrumentSerialNumber too.

Reply:
done